docs/gaia/gaiacli.md

```diff
@@ -571,3 +598,88 @@ gaiacli query gov param voting
 gaiacli query gov param tallying
 gaiacli query gov param deposit
 ```
+
+### Multisig transactions
+
+Multisig transactions require signatures of multiple private keys. Thus, generating and signing
+a transaction from a multisig account involve cooperation among the parties involved. A multisig
+transaction can be initiated by any of the keys holders, and at least one of them would need to
+import other parties' public keys into their local database and generate a multisig public key
+in order to finalize and broadcast the transaction.
+
+For example, given a multisig key comprising the keys `p1`, `p1`, and `p3`, each of which is held
+by a distinct party, the user holding `p1` would require to import both `p2` and `p3` in order to
+generate the multisig account public key:
+
+```
+gaiacli keys add \
+  --pubkey=cosmospub1addwnpepqtd28uwa0yxtwal5223qqr5aqf5y57tc7kk7z8qd4zplrdlk5ez5kdnlrj4 \
+  p2
+
+gaiacli keys add \
+  --pubkey=cosmospub1addwnpepqgj04jpm9wrdml5qnss9kjxkmxzywuklnkj0g3a3f8l5wx9z4ennz84ym5t \
+  p3
+
+gaiacli keys add \
+  --multisig=2
+  --multisig=p1,p2,p3  # note: the order matters \
+  p1p2p3
+```
+
+A new multisig public key `p1p2p3` has been stored, and its address will be
+used as signer of multisig transactions:
+
+```bash
+gaiacli keys show --address p1p2p3
+```
+
+The first step to create a multisig transaction is to initiate it on behalf
+of the multisig address created above:
+
+```bash
+gaiacli tx send \
+  --from=<multisig_address> \
+  --to=cosmos1570v2fq3twt0f0x02vhxpuzc9jc4yl30q2qned \
+  --amount=10stake \
+  --generate-only > unsignedTx.json
+```
+
+The file `unsignedTx.json` contains the unsigned transaction encoded in JSON.
+`p1` can now sign the transaction with its own private key:
+
+```bash
+gaiacli tx sign \
+  --multisig=<multisig_address> \
+  --name=p1 \
+  --output-document=p1signature.json \
+  unsignedTx.json
+```
+
+Once the signature is generated, `p1` transmits both `unsignedTx.json` and
+`p1signature.json` to `p2` or `p3`, which in turn will generate their
+respective signature:
+
+```bash
+gaiacli tx sign \
+  --multisig=<multisig_address> \
+  --name=p2 \
+  --output-document=p2signature.json \
+  unsignedTx.json
+```
+
+`p1p2p3` is a 2-of-3 multisig key, therefore one additional signature
+is sufficient. `p2` can generate the multisig transaction by combining
```

Comment:
Why `p2` specifically? Couldn't anyone run this (if they had the signature files)?

Reply:
Yes, of course. Anyone could. Amended accordingly.